klippy/extras/display/uc1701.py

```diff
@@ -19,6 +19,8 @@ def __init__(self, config):
         ppins = printer.lookup_object('pins')
         pins = [ppins.lookup_pin(config.get(name + '_pin'))
                 for name in ['cs','a0']]
+        self.contrast = config.getint('contrast',minval=0,maxval=0x3F,default=0x3F);
```

Comment:
I would recommend setting the default value for this to 0x28, which was the original defualt.  Virtually every UC1701 driver I looked at used this value.  Setting it to max contrast might negatively affect other displays.